File: app/src/main/java/com/OdiousPanda/thefweather/Model/AQI/Iaqi.java
package com.OdiousPanda.thefweather.Model.AQI;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Iaqi {

    @SerializedName("co")
    @Expose
    public Co co;
    @SerializedName("h")
    @Expose
    public H h;
    @SerializedName("no2")
    @Expose
    public No2 no2;
    @SerializedName("o3")
    @Expose
    public O3 o3;
    @SerializedName("p")
    @Expose
    public P p;
    @SerializedName("pm10")
    @Expose
    public Pm10 pm10;
    @SerializedName("pm25")
    @Expose
    public Pm25 pm25;
    @SerializedName("so2")
    @Expose
    public So2 so2;
    @SerializedName("w")
    @Expose
    public W w;

}

File: app/src/main/java/com/odiousPanda/rainbowKt/model/dataSource/aqi/Iaqi.kt
package com.odiousPanda.rainbowKt.model.dataSource.aqi


import com.google.gson.annotations.SerializedName

data class Iaqi(
    @SerializedName("co")
    val co: Co,
    @SerializedName("dew")
    val dew: Dew,
    @SerializedName("h")
    val h: H,
    @SerializedName("no2")
    val no2: No2,
    @SerializedName("o3")
    val o3: O3,
    @SerializedName("p")
    val p: P,
    @SerializedName("pm25")
    val pm25: Pm25,
    @SerializedName("so2")
    val so2: So2,
    @SerializedName("t")
    val t: T,
    @SerializedName("w")
    val w: W
)